test/unit/SLA.test.ts

```diff
@@ -175,6 +175,16 @@ describe(CONTRACT_NAMES.SLA, function () {
     expect(totalStake).equals((parseInt(mintAmount) * 2).toString());
   });
 
+  it("should distribute claiming fees to sla owner and stakeRegistry owner", async () => {
+    const { sla, dslaToken, details } = fixture;
+    let stakeAmount = 100000;
+    await dslaToken.approve(sla.address, stakeAmount);
+    await sla.stakeTokens(stakeAmount, dslaToken.address, 'long');
+    await expect(
+      sla.withdrawProviderTokens(mintAmount, dslaToken.address)
+    ).to.be.revertedWith('not finished');
+  })
```

Comment:
Doesn't seem to verify the stated assumption. Our goal is to validate that the right stakeholders receive the right amount of tokens upon a withdrawal. I understand this is more of a integration testing requirement, but this unit test is too light, still.